modules/user_manual/pages/files/access_webdav.adoc

```diff
@@ -109,31 +109,30 @@ image:files/access_webdav/dolphin-network-folder-wizard.png[Dolphin Network Fold
 
 == Creating WebDAV Mounts on the Linux Command Line
 
-You can create WebDAV mounts from the Linux command line. This is useful
-if you prefer to access ownCloud the same way as any other remote
-filesystem mount. The following example shows how to create a personal
-mount and have it mounted automatically every time you log in to your
-Linux computer.
+You can create WebDAV mounts from the Linux command line. 
+This is useful if you prefer to access ownCloud the same way as any other remote filesystem mount. 
+The following example shows how to create a personal mount and have it mounted automatically every time you boot your Linux computer.
 
-. Install the `davfs2` WebDAV filesystem driver, which allows you to
-mount WebDAV shares just like any other remote filesystem. Use this
-command to install it on Debian/Ubuntu:
+. Install the `davfs2` WebDAV filesystem driver, which allows you to mount WebDAV shares just like any other remote filesystem. 
+  Use this command to install it on Debian/Ubuntu:
 +
 ----
 sudo apt-get install davfs2
 ----
-. Use this command to install it on CentOS, Fedora, and openSUSE:
+. Use this command to install it on _CentOS_, _Fedora_, and _openSUSE_:
```

Comment:
`yum install` is centos fedora and rhel.
opensuse has `zypper install`

Testing on 15.1:

```
zypper install davfs2
Building repository 'Non-OSS Repository' cache ...........................................................................................[done]
Building repository 'Main Repository' cache ..............................................................................................[done]
Building repository 'Main Update Repository' cache .......................................................................................[done]
Building repository 'Update Repository (Non-Oss)' cache ..................................................................................[done]
Loading repository data...
Reading installed packages...
Resolving package dependencies...

The following 4 NEW packages are going to be installed:
  davfs2 fuse libexpat1 libneon27

4 new packages to install.
```
yep!

Reply:
Thanks for spotting that one. Have fixed in a local commit and will push shortly.